x-pack/plugin/security/lib/nimbus-jose-jwt-fixed/src/main/java/com/nimbusds/jose/util/JSONStringUtils.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
```

Comment:
I'm not sure if this is the right thing to do with the copyright headers with these files that are copied almost wholesale from another library.

Reply:
Normally we would retain the original copyright header, see for example FastMath.java